recipes/renode-cores/recipe.yaml

```diff
@@ -0,0 +1,89 @@
+context:
+  name: renode-cores
+  version: "1.15.3"
+  python_min: "3.6"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/renode/renode/releases/download/v${{ version }}/renode_${{ version }}_source.tar.xz
+    sha256: 40c50afe8db86a9c63f4a86b93116a6babd43bf241cfaac47b2c0d2cbc2b15a8
+  - if: unix
+    then:
+      - git: https://github.com/renode/renode-infrastructure.git
+        rev: "3fc2d5fe643068e595e875d9408cb4329522b229"
+        target_directory: cmake-renode-infrastructure
+      - git: https://github.com/antmicro/tlib.git
+        rev: "69fff75a0eba7471283b0b8db2e55e8388e284f6"
+        target_directory: cmake-tlib
```

Comment:
Why is this only cloned on Unix?  Also, instead of using `git`, let's try to downloading the archive for this revision so we can have a check sum:

```
url: https://github.com/renode/renode-infrastructure/archive/3fc2d5fe643068e595e875d9408cb4329522b229.tar.gz
url: https://github.com/antmicro/tlib/archive/69fff75a0eba7471283b0b8db2e55e8388e284f6.tar.gz
```

Reply:
Possibly a fumble, the cloning is inside the script on windows